internal/pkg/template/template.go

```diff
@@ -415,3 +436,37 @@ func (t *Test) Inputs(provider testhelper.EnvProvider, replaceEnvsFn func(string
 
 	return inputs, nil
 }
+
+func (t *Test) SaveInputs(inputs map[string]templateInput.Value) error {
+	res := make(map[string]interface{})
+	for k, v := range inputs {
+		res[k] = v.Value
+	}
+
+	// Convert to Json
+	jsonContent, err := json.EncodeString(res, true)
+	if err != nil {
+		return err
+	}
+
+	// Write file
+	f := filesystem.NewRawFile(InputsFile, jsonContent)
+	return t.fs.WriteFile(f)
+}
+
+func (t *Test) SaveExpectedOutput(prjFS filesystem.Fs) error {
+	// Get expected output dir
+	if !t.fs.IsDir(ExpectedOutDirectory) {
+		err := t.fs.Mkdir(ExpectedOutDirectory)
+		if err != nil {
+			return err
+		}
+	}
+
+	expectedFS, err := t.fs.SubDirFs(ExpectedOutDirectory)
+	if err != nil {
+		return err
+	}
+
+	return aferofs.CopyFs2Fs(prjFS, "/", expectedFS, "/")
```

Comment:
@webrouse no nic, tak ještě tohle se mi chová divně. V `prjFS` je projekt po aplikaci šablony (přes `Glob()` tam vidím description i main branch folder), ale v expectedFS není. Tam se dostane jen ten prázdný `.keboola/manifest.json`, jak jsem říkal v poledne.
![image](https://user-images.githubusercontent.com/215660/188449538-2f420d6a-556e-423b-b298-b27362a32f5a.png)

Neutušíš jak se to stane? 🙂

Reply:
Co keby si assertol/vyspisal ten `err` z `copyFs2Fs`? :)

Reply:
Ach jéje, je tam. (For the record, to ještě nejsem tak zoufalej, ale  v commitnutým kódu mám `return aferofs.CopyFsToFs()`, tak jsem čekal že ta chyba probublala nahoru, ale asi se ztratila někde v tom assertování e2e testů, tak jsem ji neviděl a myslel, že tam žádná není. 🙂)

`ERR␣&fs.PathError{Op:"chmod",␣Path:"/code/test/cli/.out/template-test-create/create/my-template/v0/tests/one/expected-out/my-template/v0/tests/one/expected-out/.keboola/manifest.json",␣Err:0x2}`

Reply:
Ale ten `chmod` už jsem v pátek někde viděl, to už se mi vracelo při nějaký jiný příležitosti. 🤔

Reply:
Přibylo tam `no such file or directory`:

`chmod /code/test/cli/.out/template-test-create/create/my-template/v0/tests/one/expected-out/my-template/v0/tests/one/expected-out/.keboola/manifest.json: no such file or directory`

Reply:
to moze byt nejaky bug v libke alebo v nasom kode ... skusim to pozriet zajtra rano

Reply:
pada mi to tiez, debugujem ....

Reply:
@JakubMatejka https://github.com/keboola/keboola-as-code/pull/795#discussion_r963516319